src/main/java/unicash/ui/StatusBarFooter.java

```diff
@@ -94,9 +93,11 @@ public void updateBalance(ObservableList<Transaction> transactions) {
         }
 
         if (Double.compare(balance, 0) < 0) {
-            balanceString = NEGATIVE_BALANCE_STRING + new Amount(Math.abs(balance));
+            double formattedNegativeBalance = Math.round((Math.abs(balance) * 100.0 / 100.0));
+            balanceString = NEGATIVE_BALANCE_STRING + String.format("%.2f", formattedNegativeBalance);
         } else {
-            balanceString = POSITIVE_BALANCE_STRING + new Amount(balance);
+            double formattedPositiveBalance = Math.round(balance * 100.0 / 100.0);
+            balanceString = POSITIVE_BALANCE_STRING + String.format("%.2f", formattedPositiveBalance);
```

Comment:
Since `%.2f` does rounding, do you still need the manual rounding?

Reply:
Oh that's right, let me get that fixed